shell/platform/android/io/flutter/plugin/platform/ImageReaderPlatformViewRenderTarget.java

```diff
@@ -55,9 +48,12 @@ protected ImageReader createImageReader33() {
     // Allow for double buffering.
     builder.setMaxImages(3);
     // Use PRIVATE image format so that we can support video decoding.
-    // TODO(johnmccutchan): Should we always use PRIVATE here? It may impact our ability
-    // to read back texture data. If we don't always want to use it, how do we decide when
-    // to use it or not? Perhaps PlatformViews can indicate if they may contain DRM'd content.
+    // TODO(johnmccutchan): Should we always use PRIVATE here? It may impact our
+    // ability
+    // to read back texture data. If we don't always want to use it, how do we
+    // decide when
+    // to use it or not? Perhaps PlatformViews can indicate if they may contain
+    // DRM'd content.
```

Comment:
nit: formatting
nit: please link an issue here explaining what has to be done (even if just what investigation is left to do or what resources someone interested in this should read about)

Reply:
Fixed added more context to the TODO.